### ISSUE TITLE
Add position: Top

### ISSUE THREAD

**Issue by wer-mathurin** [youssefali424/AnimatedRail#3]
For web something can be cool to have this rail on top.

**Comment by youssefali424** [comment]
good idea but for mobile it would be awkward, so should we different behavior between mobile and web? I think I will try to add it for both currently but it's a lot of work to do especially for the animations, I would b happy if you can contribute as well

**Comment by youssefali424** [comment]
and I think I will have to close the expansion as well, what is the point of expanding if it's position is int top or bottom
